conf/messages.en

```diff
@@ -711,6 +711,48 @@ assets.removeYesNo.title = Are you sure you want to remove the asset?
 assets.removeYesNo.heading = Are you sure you want to remove {0}?
 assets.removeYesNo.error.required = Select yes if you want to remove the asset
 
+assetAddressUkYesNo.title = Is the business asset address in the UK?
+assetAddressUkYesNo.heading = Is {0}’s address in the UK?
+assetAddressUkYesNo.checkYourAnswersLabel = Is {0} address in the UK?
+assetAddressUkYesNo.error.required = Select yes if the business asset’s address in the UK
+
+assetDescription.title = What is the description for the business asset?
+assetDescription.heading = What is the description for {0}?
+assetDescription.checkYourAnswersLabel = What is the description for {0}?
+assetDescription.hint = This must be 56 characters or less
+assetDescription.error.required = Enter a description of the business
+assetDescription.error.invalid = The description of the business asset must only include letters a to z, numbers, ampersands (&), apostrophes, back slashes, hyphens and spaces
```

Comment:
The business asset description regex is `^[0-9a-zA-Z{À-˿’}\- \&`'^]{1,56}$`

Not sure what the error message _should_ be, but this regex doesn't accept back slashes (back slashes are used in regex to escape other characters which is where the confusion may lie). It also accepts ^ characters which I believe are called _carats_.

My suggestion would be:

The description of the business asset must only include letters a to z, numbers, ampersands (&), carats (^), apostrophes, hyphens and spaces

or something like that. Worth double checking with Owen. Looks like we need to make this change for:

- other asset description
- property or land asset description
- partnership asset description

cc @adamconder

Reply:
After a conversation with Owen and adam we have agreed with your suggestion. But we will exclude carats in the error message.
The description of the business asset must only include letters a to z, numbers, ampersands (&), apostrophes, hyphens and spaces